src/components/Developers/Features.astro

```diff
@@ -0,0 +1,27 @@
+<section class="flex flex-col px-10 lg:px-20 text-center md:text-left bg-gray-100 py-24 lg:py-40">
+  <div class="m-auto">
+    <h3 class="font-black font-display text-6xl mb-16">Features</h3>
+    <div class="grid grid-cols-1 ml-0 lg:grid-cols-2 gap-y-20 w-auto text-4xl place-items-start w-full">
+      <div class="p-4 md:p-8 md:pl-0 ml-0 py-2 md:py-2 m-4 md:m-8 md:ml-0 my-2 md:my-2">
+        <h3 class="font-black font-display mb-4">Tokenization</h3>
+        <p class="text-base max-w-para">Dev Protocol makes it easy to tokenize and authenticate the ownership of the creator’s work (e.g., GitHub repository for OSS developers), and “Property tokens” are issued by each work. It works like an “Equity”, automatically monetizing and distributing rewards to its holders.</p>
+      </div>
+      <div class="p-4 md:p-8 md:pl-0 ml-0 py-2 md:py-2 m-4 md:m-8 md:ml-0 my-2 md:my-2">
+        <h3 class="font-black font-display mb-4">Staking</h3>
+        <p class="text-base max-w-para">Staking is a mechanism that increases rewards through the temporary deposit of DEV token toward a property. By using staking to trade your business, you can bring in new kinds of subscriptions that are long-term and sustainable.</p>
+      </div>
+      <div class="p-4 md:p-8 md:pl-0 ml-0 py-2 md:py-2 m-4 md:m-8 md:ml-0 my-2 md:my-2">
+        <h3 class="font-black font-display mb-4">Market</h3>
+        <p class="text-base max-w-para">Market is a contract to define the tokenizable assets in Dev Protocol, which can be added by community governance. It uses Khaos to add variety of assets in a very flexible way.</p>
+      </div>
+      <div class="p-4 md:p-8 md:pl-0 ml-0 py-2 md:py-2 m-4 md:m-8 md:ml-0 my-2 md:my-2">
+        <h3 class="font-black font-display mb-4">Khaos</h3>
+        <p class="text-base max-w-para">Khaos is a very unique oracle service designed to bring Internet data into blockchains while keeping secret information, such as secret tokens, under wraps. It can prove the ownership of the work in a secure way.</p>
```

Comment:
```suggestion
        <p class="text-base max-w-para">Khaos is a very unique oracle service designed to bring Internet data into blockchains while keeping secret information, such as secret tokens, under wraps. It can prove the ownership of the work securely.</p>
```